### DISCUSSION TITLE
Query Regarding Future WebGPU Support for the Postprocessing Project

### DISCUSSION THREAD

**Discussion by lanvada** [pmndrs/postprocessing#642] [Q&A]
I noticed in the documentation that this project does not seem to support WebGPU. Will there be consideration to add WebGPU support in the future?

**Comment by vanruesc** [comment]
WebGPU is not widely available yet, so it's not the focus right now. WebGL 1 has only recently been dropped in `three` and v7 will initially use WebGL 2. When WebGPU becomes more prevalent, there will be another transition from WebGL 2 to WebGPU.
